docs/index.md

```diff
@@ -0,0 +1,259 @@
+[Cozy](https://cozy.io) Javascript Client
+==============================
+
+`cozy-client-js` is a javascript library made by Cozy. It enables client-side  applications to make requests to the cozy stack.
+
+cozy-client-js is compatible with both cozy architectures.
+
+**Cozy-client-js is still a work-in-progress. There may be some bugs, if you find any, please open an issue.**
+
+
+Reminder about cozy architectures
+---------------------------------
+
+There is currently two actives cozy architectures :
+
+- The first, thereafter named **v2** is the existing cozy structure. It is based on 1 container / user with several node.js process in each container (cozy-data-system, cozy-proxy, cozy-home, 1 process per app).
+- The second, thereafter named **v3** is the new cozy-stack ([repository](https://github.com/cozy/cozy-stack)). It is based on go and aim to support multiple user on a single process.
+
+**v2** supported both **server-side** applications with their own node.js process and **client-side** application which only run in the user browser.
+
+**v3** will only support **client-side** application.
+
+This repository provides a library which allows you build a **client-side** application compatible with both version.
+
+The former javascript library for making client-side application was `cozy-browser-sdk`. We aim to deprecate it once the new `cozy-client-js` is feature equivalent.
+
+If you already have an application using `cozy-browser-sdk`, you can see what will change in the [transition document](./browser-sdk-transition.md). If you have any doubt, please open an issue!
+
+Include the library in your application
+----------------------
+
+You can `import`/`require` cozy-client-js using npm & webpack
+**TODO** test if `cozy-client-js` compatible with browserify, rollup, ect...
+
+You can also simply copy-paste the `dist/cozy-client.js` bundle file into your application, and include it in your application `index.html` with  `<script src="./cozy-client.js">`.
+
+
+
+Implemented API
+-----------------------
+
+# cozy.init(options)
+
+`cozy.init(options)` setup cozy-client-js and perform the necessary steps to retrieve the application token. Our goal is for you developer to not worry about how the token is retrieved, as it will depends on cozy version and context.
+
+Supported options are:
+
+ - `target` : if the cozy you want to speak to is not on the same origin than the app. Only useful for test, should not used for app installed inside Cozy.
```

Comment:
I think we should put something before the if, like "the cozy address if it is not on the same origin than the app."